Wrapping/Python/Packaging/setup.py.in

```diff
@@ -1,84 +0,0 @@
-import sys
-
-if "setuptools" in sys.modules.keys():
-    from setuptools import setup, Distribution
-else:
-    from distutils.core import setup
-    from distutils.dist import Distribution
-
-import re
-
-doc_files = @SimpleITK_DOC_FILES_AS_LIST@
-
-def get_pep386version():
```

Comment:
This versioning system is very nice. I am wondering if the SimpleITK source tree was a submodule in the SimpleITKPythonPackaging project, if the versioning could be extracted from the repo on demand, then passed to setup.py. This likely will involve calling CMake to run sitkSourceVersion.cmake